torch/testing/_internal/common_utils.py

```diff
@@ -50,16 +50,27 @@
 from enum import Enum
 from torch.autograd import gradcheck
 from torch.autograd.gradcheck import gradgradcheck
+import torch.distributed as c10d
 
 torch.backends.disable_global_flags()
 
+FILE_SCHEMA = "file://"
+if sys.platform == 'win32':
+    FILE_SCHEMA = "file:///"
+
 IS_SANDCASTLE = os.getenv('SANDCASTLE') == '1' or os.getenv('TW_JOB_USER') == 'sandcastle'
 
 class ProfilingMode(Enum):
     LEGACY = 1
     SIMPLE = 2
     PROFILING = 3
 
+def create_device(interface=None):
```

Comment:
Since this is only used in distributed test, it should go to `torch/testing/_internal/common_distributed.py` instead of this file.